make add_roles() error if result is both predictor and outcome (#1321)
tidymodels/recipes: NEWS.md

* `recipe()` will now show better error when columns are misspelled in formula (#1283).

> * `add_role()` now errors if one columns has role `"outcome"` and `"predictor"`. (#935)

Reviewer: ```suggestion
* `add_role()` now errors if a column would simultaneously have roles `"outcome"` and `"predictor"`. (#935)
```
